src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataFromFile = readDataFromFile(fromFileName);
+        String finishReport = createReport(dataFromFile);
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't creat file");
+        }
+        try {
+            Files.write(file.toPath(), finishReport.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write report to file");
+        }
```

Comment:
put this into a private method, like you've already done twice, include filenames in the messages